Source/RomanNumeral.c

```diff
@@ -6,11 +6,41 @@
 #include <stdio.h>
 #include "RomanNumeral.h"
 
-static char * Append(char *output, const char *toAppend)
+static char *Append(char *output, const char *toAppend)
 {
    return output + sprintf(output, "%s", toAppend);
 }
 
+typedef struct
+{
+   int num;
```

Comment:
`int` is probably fine for practicing, but be aware that we would like to see this be using `stdint.h`. So `uint16_t` or `uint32_t` would probably work here.